src/utils/ucx/ucx_utils.cpp

```diff
@@ -28,9 +28,7 @@
 #include "ucx_utils.h"
 #include "common/nixl_log.h"
 
-using namespace std;
```

Comment:
maybe it is possible to use something like
`using std::size_t;`
and maybe some other types?

Reply:
ok